sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -822,6 +822,8 @@ object FunctionRegistry {
     expression[ParseJson]("parse_json"),
     expressionBuilder("variant_get", VariantGetExpressionBuilder),
     expressionBuilder("try_variant_get", TryVariantGetExpressionBuilder),
+    expression[VariantExplode]("variant_explode"),
+    expressionGeneratorOuter[VariantExplode]("variant_explode_outer"),
```

Comment:
It's not recommended to use the hive style generator SQL syntax anymore. Can we remove it and only register it as TVF?

Reply:
Done.